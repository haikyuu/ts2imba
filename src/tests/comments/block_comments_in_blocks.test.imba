import {build} from "../../index"
import * as imbac from 'imba/compiler'

let tsx-code = '''
if (x) {
  /*
   * hello
   * world
   */
  y();
}
'''

test 'block_comments_in_blocks' do
	const result = await build tsx-code
	expect(result.code).toMatchSnapshot()
	const out = imbac.compile(result.code, sourceId: 'sth')
	expect(out.js).toBeDefined()
