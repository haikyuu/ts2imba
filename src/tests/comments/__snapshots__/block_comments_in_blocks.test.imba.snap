// Vitest Snapshot v1

exports[`block_comments_in_blocks 1`] = `
"if x
	###
# hello
# world
###	y()
"
`;
